python/src/equistore/operations/ones_like.py

```diff
@@ -1,6 +1,7 @@
 from typing import List, Union
+import numpy as np
```

Comment:
The linter seems not to be soo smart, so it throws an error here
https://github.com/lab-cosmo/equistore/actions/runs/4417733640/jobs/7745861938#step:7:532
In the code you don't need numpy, so it complains. It apparently cannot see that it is used in the doc.
This should fix the problem
```suggestion
import numpy as np # noqa: F401
```

Reply:
Okay talked with @DavideTisi and @Luthaf we move this import to the docs, so you can remove the import here and insert it into the docstring
```suggestion
```